### PR TITLE
I Korolev 

### PR DESCRIPTION
**Pull request by Korolev731**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,4 @@ Main purpose of this repository is collaboration of homework for study groups
 ## CI/CD
 
 In preparing
+# 1
```
Comment (pluhin) [inline README.md:19]: Please remove this changes

File: readme.md [hunk 2]
```diff
@@ -0,0 +1 @@
+##1 commit
```
Comment (pluhin) [inline readme.md:1]: Please remove this file

**Review by pluhin — CHANGES_REQUESTED** [review]

File: readme.md (hunk 2 above)
Comment (Korolev731) [reply readme.md]: fixed

File: README.md (hunk 1 above)
Comment (Korolev731) [reply README.md]: fixed

**Review by pluhin — CHANGES_REQUESTED** [review]
Do not remove not your files. It is not good for shared project

**Pull request closed without merging**